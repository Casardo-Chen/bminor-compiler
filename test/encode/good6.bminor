"To\tBe\tOr\tNot\tTo\tBe"