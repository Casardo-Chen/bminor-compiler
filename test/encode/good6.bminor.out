Input String    : "To\tBe\tOr\tNot\tTo\tBe"
Decoded String    : To	Be	Or	Not	To	Be
Output String    : "To\tBe\tOr\tNot\tTo\tBe"
